internal/providers/amazon/objectstore.go

```diff
@@ -23,9 +23,14 @@ type bucketNotFoundError struct{}
 func (bucketNotFoundError) Error() string  { return "bucket not found" }
 func (bucketNotFoundError) NotFound() bool { return true }
 
+type amazonObjectStore interface {
+	commonObjectstore.ObjectStore
+	GetRegion(bucket string) (string, error)
```

Comment:
This should be private if not used by other packages, only inside the amazon objectstore.

Reply:
I think no interface should have private methods.